client/styles/SmarterBalancedStyles/styles/custom.less

```diff
@@ -257,12 +257,12 @@ h1, h2, h3, h4, h5 {
 }
 
 .selected {
```

Comment:
this .selected class could be conflicting with score-page.less .selected

Reply:
changes with the smarter balanced styles should probably me made in their own [repo](https://github.com/osu-cass/SmarterBalancedStyles)